internal/provider/resource_basistheory_proxy.go

```diff
@@ -0,0 +1,183 @@
+package provider
+
+import (
+	"context"
+	"github.com/Basis-Theory/basistheory-go/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceBasisTheoryProxy() *schema.Resource {
+	return &schema.Resource{
+		Description: "Proxy https://docs.basistheory.com/#proxies",
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		CreateContext: resourceProxyCreate,
+		ReadContext:   resourceProxyRead,
+		UpdateContext: resourceProxyUpdate,
+		DeleteContext: resourceProxyDelete,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Description: "Unique identifier for the Proxy",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"key": {
```

Comment:
Should this be considered sensitive?

Reply:
I suppose we did move that from the URL to a header now 👍